CONTRIBUTING.md

```diff
@@ -54,6 +54,26 @@ Signed-off-by: Your Name <YourName@example.org>
 in a single line, into the commit comment field. This can be automated by using a browser plugin like
 [DCO GitHub UI](https://github.com/scottrigby/dco-gh-ui).
 
+#### Steps to use the DCO browser plugin
+- Go to the plugin page on the [chrome web store](https://chrome.google.com/webstore/detail/dco-github-ui/onhgmjhnaeipfgacbglaphlmllkpoijo).
```

Comment:
Include link to github repo in addition to the plugin pages, just in case plugin pages change URL

Reply:
I didn't add the link since it's mentioned just above that on line 55. 
I'll add it if you want

Reply:
Done

Reply:
Thanks

Reply:
> @GeorgLink , I didn't add the link since it's mentioned just above that on line 55. But I'll add it anyway

I thought as much. It looks good now, especially when someone looks at only this section, they know where to get information about the bot.